cargo-dist-schema/src/snapshots/cargo_dist_schema__emit.snap

```diff
@@ -772,7 +772,7 @@ expression: json_schema
           ]
         },
         "runner": {
-          "description": "Github Runner to user",
+          "description": "Github Runner to use",
```

Comment:
It does, I can get rid of it if needed? But I would hate for such a small thing to block landing this

Reply:
What I mean is the test failure is because of this - this is a snapshot change without a corresponding code change. I think you had probably edited the comment in a previous commit in the PR you extracted this from, then committed the snapshot change in this commit.